src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/index.tsx

```diff
@@ -116,6 +101,7 @@ class Breadcrumbs extends React.Component<Props, State> {
       filteredByFilter: tranformedCrumbs,
       filteredBySearch: this.getCollapsedBreadcrumbs(tranformedCrumbs),
       filterOptions,
+      relativeTime: tranformedCrumbs[tranformedCrumbs.length - 1]?.timestamp,
```

Comment:
Should this be `transformedCrumbs`? You are currently missing an s.

Reply:
sure..good catch!!